packages/@aws-cdk/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -183,40 +183,35 @@ export class AthenaStartQueryExecution extends sfn.TaskStateBase {
    * @internal
    */
   protected _renderTask(): any {
+
+    let renderedObject = {
```

Comment:
prefer using `const` over `let` unless specifically unnecessary
```suggestion
    const renderedObject = {
```